client/src/components/NewBoardGameForm.js

```diff
@@ -62,6 +67,12 @@ const NewBoardGameForm = props => {
 		addNewBoardGame()
 	}
 
+	useEffect(() => {
+		if (role === "user" || role === "admin") {
+			setShowButton(true)
+		}
+	})
```

Comment:
same comment as your BoardGameShowPage. We dont need state or useEffect to conditionally render the button

Talk about it with your group if you need support, or ask a question in the queue